ansible/roles/caddy/templates/caddyfile.j2

```diff
@@ -0,0 +1,19 @@
+{
+  skip_install_trust
+
+  log {
+    format filter {
+      wrap json
+      fields {
+        common_log delete
+        request>headers>Authorization delete
```

Comment:
Caddy logs every single header by default. It's probably better than Nginx (that logs just few) since you only need them when you **need** them. The drawback is that we need to ensure secrets are not exposed. `Authorization` header is the only one that contains secrets I know about. Can you think of anything else?

Reply:
Nope! This should be fine!